backend/clubs/views.py

```diff
@@ -3088,6 +3099,13 @@ def email_blast(self, request, *args, **kwargs):
         officer_emails = event.club.get_officer_emails()
         emails = list(holder_emails) + list(officer_emails)
 
+        content = request.data.get("content").strip()
```

Comment:
Might error out if no content is provided before being able to return the response, maybe request.data.get("content", "")?